charts/deploy-tmp-notebooks.py

```diff
@@ -110,6 +120,10 @@ def make_tmp_values(values, renku_namespace, out_path=None):
     # Add some reasonably short defaults for server culling
     new_values["jupyterhub"]["cull"] = {"enabled": True, "timeout": 3600, "every": 60}
 
+    # Disable user userScheduler
+    new_values["jupyterhub"]["scheduling"]["userScheduler"] = {"enabled": False }
```

Comment:
I guess this will not be needed after https://github.com/SwissDataScienceCenter/renku-notebooks/pull/429?

Reply:
yes, I'll do a small PR to cleanup this and the main Renku values after merging that PR into this repo.

Reply:
I'm not sure you need to bother. Having those deployed for another day won't hurt anything and it's just extra work for you. It's a problem that will correct itself soon.

Reply:
what I'm saying is: just remove these two lines :)

Reply:
if I remove the two lines the checks would not pass

Reply:
I mean the PRs would not be able to deploy because they are not using Tiller which had more permissions than renkubot

Reply:
Oh got it because they need service accounts made 👍